src/components/UserProfile/TeamsAndProjects/UserProjectsTable.jsx

```diff
@@ -294,7 +297,10 @@ const UserProjectsTable = React.memo(props => {
               <span className="projects-span">Projects</span>
             </Col>
             {props.edit && props.role && (
-              <Col md="5" style={{ display: 'flex', alignItems: 'center', justifyContent: 'center'}}>
+              <Col
+                md="5"
+                style={{ display: 'flex', alignItems: 'center', justifyContent: 'center' }}
```

Comment:
As pointed in one of the previous comments, maybe create a separate class with common styles so it is reusable.